tests/integration/expected_test_results/api_v3_taxonomy_suggestions/categories-term-strawberry.json

```diff
@@ -2,6 +2,7 @@
    "errors" : [],
    "status" : "success",
    "suggestions" : [
+      "Mint-flavoured syrup with sugar diluted in water",
```

Comment:
This entry is because we have a weird CIQUAL category that applies to both strawberry and mint flavoured syrups... To be fixed in the taxonomy.